SIA.tex

```diff
@@ -506,7 +496,7 @@ \subsubsection{UPLOAD}
  The DALI UPLOAD parameter is not used by this version of SIA. The use case of uploading lists of coordinates is covered by the multiple-valued parameters values.
 
 \subsubsection{Service PARAMETER self description}
-Any service may  include a DataLink service descriptor in the VOTable output to describe itself. This descriptor would describe the supported query parameters (standard and custom), including list of values for those with a fixed list (e.g. COLLECTION, INSTRUMENT, FACILITY, DPTYPE, CALIB, and FORMAT).
+Any service SHOULD  include a DataLink service descriptor in the VOTable output to describe itself. This descriptor would describe the supported query parameters (standard and custom), including list of values for those with a fixed list (COLLECTION, INSTRUMENT, FACILITY, DPTYPE, CALIB, and FORMAT). This is peculiarly important for those parameters where the fixed list is not standardized by any specification such as COLLECTION, INSTRUMENT and FACILITY.
```

Comment:
I think `peculiarly` should be `particularly`

Reply:
This change has been made (although I'm not able to distinguish the meaning of the two words)